src/api/index.ts

```diff
@@ -187,3 +188,20 @@ export async function getRecentBlocks(
   }
   return blocks;
 }
+
+export async function getStake(
+  nodeUrl: string,
+  delegatorAddress: Types.Address,
+): Promise<Types.MoveValue[]> {
+  const client = new AptosClient(nodeUrl);
```

Comment:
not sure we want create a new instance every time we call this function (which seems like it is every time stake or state is changing) - ideally we would have one `client` instance in our app that changes when the `network` changes (maybe a good place for it is in global state).

Sounds it is out of this PR scope, but something to look at

Reply:
WP-325 linear issue created!